published_versions.json

```diff
@@ -189,13 +226,256 @@
               "Download here (Debug version):": "https://download.pytorch.org/libtorch/nightly/cu113/libtorch-win-shared-with-deps-debug-latest.zip"
             }
           },
+          "cuda11.6": {
+            "note": null,
+            "versions": {
+              "Download here (Release version):": "https://download.pytorch.org/libtorch/nightly/cu116/libtorch-win-shared-with-deps-latest.zip",
+              "Download here (Debug version):": "https://download.pytorch.org/libtorch/nightly/cu116/libtorch-win-shared-with-deps-debug-latest.zip"
+            }
+          },
           "rocm4.x": {
             "note": "<b>NOTE:</b> ROCm is not available on Windows",
             "versions": null
           }
         }
       }
     },
+    "1.12.0": {
+      "linux": {
+        "pip": {
+          "accnone": {
+            "note": null,
+            "command": "pip3 install torch==1.12.0+cpu torchvision==0.13.0+cpu -f https://download.pytorch.org/whl/test/cpu/torch_test.html"
```

Comment:
Can we use `--extra-index-urls` here (same as in preview?)

Reply:
In this case we can not use --extra-index-urls

```
pip3 install torch==1.11.0+cu113 --extra-index-url https://download.pytorch.org/whl/test/cu113/torch_test.html
Looking in indexes: https://pypi.org/simple, https://download.pytorch.org/whl/test/cu113/torch_test.html
ERROR: Could not find a version that satisfies the requirement torch==1.11.0+cu113 (from versions: 1.7.1, 1.8.0, 1.8.1, 1.9.0, 1.9.1, 1.10.0, 1.10.1, 1.10.2, 1.11.0)
ERROR: No matching distribution found for torch==1.11.0+cu113
(env1) atalman@atalman-2217-764688c9d7-w4j6c:~$ pip3 install torch==1.11.0 --extra-index-url https://download.pytorch.org/whl/test/cu113/torch_test.html
Looking in indexes: https://pypi.org/simple, https://download.pytorch.org/whl/test/cu113/torch_test.html
Collecting torch==1.11.0
  Downloading torch-1.11.0-cp39-cp39-manylinux1_x86_64.whl (750.6 MB)
     |████████████████████████████████| 750.6 MB 7.9 kB/s 
Requirement already satisfied: typing-extensions in ./conda/envs/env1/lib/python3.9/site-packages (from torch==1.11.0) (4.1.1)
Installing collected packages: torch
  Attempting uninstall: torch
    Found existing installation: torch 1.12.0.dev20220325
    Uninstalling torch-1.12.0.dev20220325:
      Successfully uninstalled torch-1.12.0.dev20220325
```

Reply:
This should work:
```
pip3 install torch==1.11.0+cu113 --extra-index-url https://download.pytorch.org/whl/test/cu113/
```

Reply:
Done! using extra-index-url everywhre